RELEASE.md

```diff
@@ -35,6 +35,22 @@ The aptos framework release tags are created to facilitate the on-chain framewor
 ### aptos-cli-v*X.Y.Z* release tag
 The aptos cli release tags are created to track the CLI versions for community to use when developing on the Aptos network. It's always recommended to upgrade your CLI when a new version is released, for the best user experience. Learn how to update to the [latest CLI version](https://aptos.dev/tools/install-cli/install-from-brew/#upgrading-the-cli).
 
+## Aptos Release Lifecycle
+(The time length here is a rough estimate, it varies depends on each release.)
+* [day 0] A release branch `aptos-release-vx.y` will be created, with a commit hash `abcde`. The full test suite will be triggered for the commit hash for validation.
+* [day 1] The release will be deployed to **devnet**.
+* [day 7] Once the release passed devnet test, a release tag `aptos-release-vx.y.z` will be created, and get deployed to **testnet**.
```

Comment:
correction: release tag is called `aptos-node-vx.y.z`, though the release branch is called `aptos-release-vx.y`. Maybe add a note somewhere that this is for the `aptos-node` release specifically, and that CLI etc have their own release cadences?